### PR TITLE
Select tags when creating triggers

### PR DESCRIPTION
**Pull request by shairyar**
Customers usually miss selecting tags when creating a trigger, this results in triggers not generating alerts. The docs have been updated to mention the selection of tags is important.

### CODE REVIEW

File: source/anomaly-detection/index.html.md [hunk 1]
```diff
@@ -48,6 +48,13 @@ Triggers can be configured for a variety of metrics, such as, but not limited to
   - Memory usage
   - Network usage
 
+### Tags
+Some metrics requires tags to be selected as well. If a metric has tags AppSignal will let you know, for example in the following trigger `action_mailer_process` metric was selected and AppSignal shows for this metric `action` and `mailer` tag combinations must be applied in order for the trigger to work correctly.
```
Comment (matsimitsu) [inline source/anomaly-detection/index.html.md:52]: Still a bit confusing, I think. 

> If tags were added when sending a metric to AppSignal, all tag keys must match in order to generate an alert. 

E.g. if a metric was tagged with `:namespace => "web", :action => "Homepage"`, both the `namespace` and `action` tags must be present in the anomaly detection form.

Comment (shairyar) [reply source/anomaly-detection/index.html.md:52]: Let's expand on that example you gave, suppose I am sending the following metric to AppSignal

```
Appsignal.set_gauge("my_metric_name", 100, : namespace => "web", :action => "web")

```

Then while creating a trigger I need to apply both tags (`namespace` and `web`)

Comment (matsimitsu) [reply source/anomaly-detection/index.html.md:52]: yes

Comment (shairyar) [reply source/anomaly-detection/index.html.md]: I have improved the example.

File: source/anomaly-detection/index.html.md [hunk 2]
```diff
@@ -48,6 +48,19 @@ Triggers can be configured for a variety of metrics, such as, but not limited to
...
+Some metrics require tags to be selected as well. For example if you are sending a custom metric to AppSignal that has tags:
+
+
+```
+Appsignal.set_gauge("my_metric_name", 100, : namespace => "web", :action => "hello")
+```
+
+Then while creating a trigger for this metric you need to select tags as well.
+
+<img src="/assets/images/screenshots/trigger-with-tags.png">
```
Comment (matsimitsu) [inline source/anomaly-detection/index.html.md:61]: Could you use the markdown version of this tag? `Trigger with tags`

**Review by matsimitsu — CHANGES_REQUESTED** [review]

File: source/anomaly-detection/index.html.md (hunk 2 above)
Comment (shairyar) [reply source/anomaly-detection/index.html.md]: Done

**Review by matsimitsu — APPROVED** [review]

**Pull request merged**